conf/st2.tests2.conf

```diff
@@ -0,0 +1,95 @@
+# Config file used by integration tests
```

Comment:
Why is there a tests2.conf?

Reply:
Sorry - I started drafting an issue to explain this but it got lost in my tabs. Here's an explanation: https://github.com/StackStorm/st2/issues/3775. I added a reference in this config to that issue in 7c5fe270f so future us know why it exists.

Reply:
We can merge for now. Please add some comments into st2test2.conf why it's needed. Follow up on the issue you referenced here until it's closed.

Reply:
Yup, done in https://github.com/StackStorm/st2/pull/3762/commits/7c5fe270fc75653e03cf6b1fd30680c4c844270a. Will add a TODO to that issue to remove this config and allow the inquiry gc tests to use `st2.tests.conf`

Reply:
Sorry for nitpicking. I like to see additional comments at the top of st2.tests2.conf. No one will be able to identify that the comments at line xyz explain why this file is needed.

Reply:
No worries, added in e0b1dfbf8